DesktopEdge/packages.config

```diff
@@ -1,6 +0,0 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
-<packages>
-  <package id="Newtonsoft.Json" version="12.0.3" targetFramework="net48" />

```

Comment:
were these removed on purpose? i would expect the DesktopEdge to need these

Reply:
Jeremy upgraded the package config. @JeremyTellier  Can you answer this question?

Reply:
Package are not handled that way anymore that is legacy